src/gc/windows/gchelpers_amd64.asm

```diff
@@ -0,0 +1,93 @@
+
+
+push_nonvol_reg macro Reg
```

Comment:
You do not really need this pile assembly code to call cpuid. Recent versions of Visual C++ have intrisics for cpuid - https://msdn.microsoft.com/en-us/library/hskdteyh.aspx